README.md

```diff
@@ -1,123 +1,323 @@
-4. [Validate Dataset in BigQuery and Re-identify using Dataflow](https://cloud.google.com/solutions/validating-de-identified-data-bigquery-re-identifying-pii-data).   
-
-## Table of Contents  
-* [Quick Start](#quick-start).  
-	 
-	* [Reference Architecture](#reference-architecture).  
-	* [Quick Start- Setup Data Tokenization Demo](#quick-start).    	
-
-* [V2 Solution By Using In Built Java Beam Transform](#v2-solution-by-using-in-built-java-beam-transform).  
-   
-
-## Reference Architecture
-![Reference Architecture](diagrams/ref_arch_solution.png)	   	
-
-## Quick Start
-[![Open in Cloud Shell](http://gstatic.com/cloudssh/images/open-btn.svg)](https://console.cloud.google.com/cloudshell/editor?cloudshell_git_repo=https://github.com/GoogleCloudPlatform/dlp-dataflow-deidentification.git)
-
-  Run the following commands to trigger an automated deployment in your GCP project. Script handles following topics:   
- 
- * Create a bucket ({project-id}-demo-data) in us-central1 and [uploads a sample dataset](https://cloud.google.com/solutions/creating-cloud-dlp-de-identification-transformation-templates-pii-dataset#downloading_the_sample_files) with <b>mock</b> PII data.  
- 
- * Create a BigQuery dataset in US (demo_dataset) to store the tokenized data.  
- 
- * Create a [KMS wrapped key(KEK)](https://cloud.google.com/solutions/creating-cloud-dlp-de-identification-transformation-templates-pii-dataset#creating_a_key_encryption_key_kek) by creating an automatic [TEK](https://cloud.google.com/solutions/de-identification-re-identification-pii-using-cloud-dlp#token_encryption_keys) (Token Encryption Key). 
-  
- * Create DLP [inspect and re-identification template](https://cloud.google.com/solutions/creating-cloud-dlp-de-identification-transformation-templates-pii-dataset#creating_the_cloud_dlp_templates) with the KEK and crypto based transformations identified in this [section of the guide](https://cloud.google.com/solutions/de-identification-re-identification-pii-using-cloud-dlp#determining_transformation_type)
-  
- * Trigger an [automated Dataflow pipeline](https://cloud.google.com/dataflow/docs/guides/templates/provided-streaming#data-maskingtokenization-using-cloud-dlp-from-cloud-storage-to-bigquery-stream) by passing all the required [parameters](https://cloud.google.com/solutions/running-automated-dataflow-pipeline-de-identify-pii-dataset#reviewing_the_pipeline_parameters) e.g: data, configuration & dataset name.  
- 
- * Please allow 5-10 mins for the deployment to be completed.
+
+# Inspect, De-identify and Re-identify sensitive data using Cloud DLP and Dataflow
+
+> This repo contains a reference implementation of an end to end data tokenization solution. The solution is designed to migrate sensitive data to BigQuery after passing it through inspection/de-identification/re-identification Dataflow pipelines implemented using Cloud DLP. Please check out the links below for reference guides.
```

Comment:
We can call it out to be 
"End to End data tokenization & inspection solution" as tokenization means -
"Tokenization, when applied to data security, is the process of substituting a sensitive data element with a non-sensitive equivalent,